packages/ingest/test/test-sftp.js

```diff
@@ -87,3 +90,28 @@ test('Download remote file to s3 with correct content-type', async (t) => {
   const s3HeadResponse = await headObject(bucket, key);
   t.is(expectedContentType, s3HeadResponse.ContentType);
 });
+
+test('Download remote file to local disk', async (t) => {
+  class MyTestSftpDiscoveryClass extends TestSftpMixin(MyTestDiscoveryClass) {}
+  const myTestSftpDiscoveryClass = new MyTestSftpDiscoveryClass(true);
+
+  const localPath = path.join(os.tmpdir(), `delete-me-${randomString()}.txt`);
+  await myTestSftpDiscoveryClass.download(
+    '/granules/MOD09GQ.A2017224.h27v08.006.2017227165029.hdf', localPath
+  );
+
+  const sum = await generateChecksumFromStream('CKSUM', fs.createReadStream(localPath));
+  t.is(sum, 1435712144);
+  fs.unlinkSync(localPath);
+});
+
+test('Write data to remote file', async (t) => {
+  class MyTestSftpDiscoveryClass extends TestSftpMixin(MyTestDiscoveryClass) {}
+  const myTestSftpDiscoveryClass = new MyTestSftpDiscoveryClass(true);
+
+  await myTestSftpDiscoveryClass.write(
+    '/granules', 'delete-me-test-file', 'mytestdata'
+  );
+  t.pass();
```

Comment:
should this test that the file was written?

Reply:
yes. added checksum